example_dags/example_dag_fivetran.py

```diff
@@ -11,6 +11,9 @@
 
 s3_bucket = os.getenv("S3_BUCKET", "s3://astro-sdk-test")
 gcs_bucket = os.getenv("GCS_BUCKET", "gs://uto-test")
+snowflake_database = os.getenv("SNOWFLAKE_DATABASE", "dummy-database")
```

Comment:
Why we need these changes in this file?

Reply:
When someone uses the older version of this example DAG as it is, it would throw an error as the environment variable would not be set.

Also, both the DAGs in the example-dags folder had the same name.